### PR TITLE
windows dual compiler cmake build & test

### PR DESCRIPTION
**Pull request by TorreZuk**
* windows build and test

### CODE REVIEW

File: common/helpers.hpp [hunk 1]
```diff
@@ -129,15 +132,32 @@ namespace helpers
         std::random_device                 rd{};
         std::mt19937                       gen{rd()};
         std::uniform_int_distribution<int> distrib{-range, range};
-        distrib(gen); // prime generator to remove warning
+        (void)distrib(gen); // prime generator to remove warning
```
Comment (rkamd) [inline common/helpers.hpp:135]: Just curious, what warning does this line resolve?

Comment (TorreZuk) [reply common/helpers.hpp:135]: You mean the change (void) for unused result, or the prime generator?   std::uniform_int_distribution is same as rocblas usage would have to look at larger context.

**Review by NaveenElumalaiAMD — APPROVED** [review]
LGTM. Do we have to add the build instructions in the README.md?

**Comment by TorreZuk** [comment]
> LGTM. Do we have to add the build instructions in the README.md?

Eventually, but for now just getting it all set up.

**Pull request merged**